src/Orleans.Runtime/Hosting/CoreHostingExtensions.cs

```diff
@@ -16,6 +18,32 @@ public static class CoreHostingExtensions
         /// <param name="builder">The host builder.</param>
         /// <param name="siloName">The silo name.</param>
         /// <returns>The silo builder.</returns>
+        public static ISiloHostBuilder ConfigureOrleans(this ISiloHostBuilder builder)
+        {
+            // Configure the container to use Orleans, including the default silo name & services.
+            builder.ConfigureServices((context, services) =>
+            {
+                if (!context.Properties.ContainsKey("OrleansServicesAdded"))
```

Comment:
Is this pattern also used in ASP.NET?

Reply:
Not really, this is because until we move to the generic HostBuilder, I wanted to have a way to opt-out from adding the services, so that I can test the SiloHostBuilder in isolation from Orleans, to verify it aligns with the other one.
In the generic host builder, if the user doesn't explicitly call `ConfigureWebHost()`, the host builder doesn't include anything ASP.NET related